sharktank/sharktank/utils/testing.py

```diff
@@ -257,16 +264,10 @@ def print_stats(label, t):
         # Then assert.
         for name in ref_f.keys():
             actual = actual_f.get_tensor(name)
+            if allow_different_dtype:
+                actual = actual.to(dtype=ref.dtype)
```

Comment:
Why not use `check_dtype` arg for `assert_close`?

Reply:
Done.